server/src/db/index.ts

```diff
@@ -0,0 +1,48 @@
+import mongoose, {Schema} from 'mongoose';
+
+// Recruiter accounts
+let UserSchema : Schema = new Schema({
+  name: String,
+  email: String,
+  password: String
+})
+
+// Questions
+let QuestionSchema : Schema = new Schema({
+  prompt: String,
+  options: [String],
+  type: String
```

Comment:
we can make this an enum later

Reply:
Yeah will do that for the final version of this PR